packages/solutions/app-tools/src/analyze/utils.ts

```diff
@@ -169,7 +169,14 @@ export const getServerCombinedModueFile = (
 };
 
 export const checkIsBuildCommands = () => {
-  const buildCommands = ['dev', 'start', 'build', 'inspect', 'deploy'];
+  const buildCommands = [
+    'dev',
+    'start',
+    'build',
+    'inspect',
+    'deploy',
+    'dev-worker',
```

Comment:
this command is not exist in modern.js...maybe we can optimize the logic here later

Reply:
yeap, Right!

Reply:
We can do this in other PR.